templates/hazelcast.xml.erb

```diff
@@ -0,0 +1,287 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<hazelcast xmlns="http://www.hazelcast.com/schema/config"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://www.hazelcast.com/schema/config hazelcast-config-3.6.xsd">
+  <group>
+    <name>nexus</name>
+    <password>nexus123</password>
+  </group>
+
+  <management-center enabled="true">http://localhost:8080/</management-center>
+
+  <properties>
+    <property name="hazelcast.shutdownhook.enabled">false</property>
+    <property name="hazelcast.logging.type">slf4j</property>
+    <property name="hazelcast.jmx">true</property>
+    <!--<property name="hazelcast.jmx.detailed">false</property>-->
+    <property name="hazelcast.phone.home.enabled">false</property>
+    <property name="hazelcast.prefer.ipv4.stack">true</property>
+    <property name="hazelcast.health.monitoring.level">SILENT</property>
+
+    <!-- OrientDB: use low number of threads -->
+    <property name="hazelcast.operation.thread.count">1</property>
+    <property name="hazelcast.operation.generic.thread.count">1</property>
+    <property name="hazelcast.client.event.thread.count">1</property>
+    <property name="hazelcast.event.thread.count">1</property>
+
+    <!-- OrientDB: reduce split-brain check interval -->
+    <property name="hazelcast.max.no.heartbeat.seconds">30</property>
+    <property name="hazelcast.heartbeat.interval.seconds">5</property>
+
+    <!-- OrientDB: reduce split-brain merge interval -->
+    <property name="hazelcast.merge.next.run.delay.seconds">15</property>
+
+    <!-- OrientDB: use ICMP to detect crashed nodes -->
+    <property name="hazelcast.icmp.enabled">true</property>
+    <property name="hazelcast.icmp.timeout">5000</property>
+    <property name="hazelcast.icmp.ttl">3</property>
+
+    <property name="hazelcast.discovery.enabled"><%= node['nexus_repository_manager']['properties']['hazelcast']['discovery']['enabled'] %></property>
+  </properties>
+
+  <network>
+    <port auto-increment="true" port-count="100">5701</port>
+
+    <outbound-ports>
+      <!--
+          Allowed port range when connecting to other nodes.
+          0 or * means use system provided port.
+      -->
+      <ports>0</ports>
+    </outbound-ports>
+
+    <join>
+      <multicast enabled="<%= node['nexus_repository_manager']['hazelcast']['network']['join']['multicast']['enabled'] %>">
+        <multicast-group><%= node['nexus_repository_manager']['hazelcast']['network']['join']['multicast']['multicast_group'] %></multicast-group>
+        <multicast-port><%= node['nexus_repository_manager']['hazelcast']['network']['join']['multicast']['multicast_port'] %></multicast-port>
+      </multicast>
+      <tcp-ip enabled="false">
```

Comment:
this should either become an attribute and the iterated over the IP addresses, or removed.

Reply:
It is now and attribute and we iterate over the members.